resources/assets/js/subscription.js

```diff
@@ -26,18 +26,6 @@ function initStripe() {
     elements = stripe.elements();
 }
 
-function initCancelListener()
```

Comment:
Great but `initCancelListener()` is still being called from another part of this script :)